content/docs/intro/cloud-providers/okta/_index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Okta
+meta_desc: This page provides an overview of the Okta Provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-okta
+    weight: 5
+---
+
+<img src="/logos/tech/okta.png" align="right" class="h-16 px-8 pb-4">
+
+The Okta provider for Pulumi can be used to provision any of the resources available in [Okta](https://www.okta.com/).
+
+See the [full API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/okta" >}}) for complete details of the available Okta provider APIs.
+
+## Setup
+
+The Okta provider supports several options for interacting with Okta.  See the [Okta setup page]({{< relref "setup.md" >}}) for details.
```

Comment:
I just [removed](https://github.com/pulumi/docs/pull/2196) the ".md" from all our `relref` links, as Hugo doesn't require it.

```suggestion
The Okta provider supports several options for interacting with Okta.  See the [Okta setup page]({{< relref "setup" >}}) for details.
```